k8s/models/job.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python
+# -*- coding: utf-8
+from __future__ import absolute_import
+
+from .pod import PodTemplateSpec
+from .common import ObjectMeta
+from ..base import Model
+from ..fields import Field
+
+
+class JobSpec(Model):
+    template = Field(PodTemplateSpec)
+    backoffLimit = Field(int)
```

Comment:
Should we define a few more of the fields here? `activeDeadlineSeconds`, `completions` and `parallelism` seems like fields that are going to be commonly used.

Reply:
Done